### PR TITLE
A few small changes here and there

### PR DESCRIPTION
**Pull request by bgruening**

### CODE REVIEW

**Review by wm75 — APPROVED** [review]
Thanks!

**Comment by bedroesb** [comment]
@bgruening  and @wm75 is this WIP? Tell me when I should merge things, or can help out. Eventually we will have to make a PR with this content to the real one!

File: showcase/covid19-galaxy.md [hunk 1]
```diff
@@ -49,31 +55,37 @@ The [WorkflowHub registry](https://workflowhub.eu/) for scientific workflows pro
...
 ### Viral Beacon project
 
-In the showcase the [Viral Beacon project](https://covid19beacon.crg.eu/) acts as a consumer of the data and provides visualisations for it.
+In the showcase the [Viral Beacon project](https://covid19beacon.crg.eu/) acts as a consumer of the data and provides visualisations.
 
 ### UCSC genome browser
 
-The genome browser offers a [Galaxy ENA mutations](https://genome.ucsc.edu/cgi-bin/hgTrackUi?db=wuhCor1&c=NC_045512v2&g=galaxyEna) track populated from the data on the CRG’s server.
+The genome browser offers a [Galaxy ENA mutations](https://genome.ucsc.edu/cgi-bin/hgTrackUi?db=wuhCor1&c=NC_045512v2&g=galaxyEna)
+track populated from the data on the CRG’s archive.
```
Comment (bedroesb) [inline showcase/covid19-galaxy.md:80]:
```suggestion
track populated from the data in the CRG’s archive.
```

**Comment by bedroesb** [comment]
Great, thanks!

**Pull request merged**